_posts/2013-05-15-understanding-vestibular-disorders.md

```diff
@@ -12,23 +12,23 @@ further_reading:
     source: Val Head
     year: 2015
 ---
->**Vestibular system** — n
->The sensory mechanism in the inner ear that detects movement of the head and helps to control balance.
 
-Imagine a world where your internal gyroscope is not working properly. Very similar to being intoxicated, things seem to move of their own accord, your feet never quite seem to be stable underneath you, and your senses are moving faster or slower than your body.
+Your Vestibular system us the sensory mechanism in the inner ear that detects movement of the head and helps to control balance.
```

Comment:
"us" => "is"

Reply:
Fixed in https://github.com/a11yproject/a11yproject.com/commit/6c8178f28780632d9213bc611615fd88a6e5ce66.